src/app/playlists/[id]/page.tsx

```
@@ -0,0 +1,128 @@
+            {tracks.map((track) => (
+              <button
+                key={track.track_id}
+                onClick={() => handlePlay(track)}
+                className="bg-gray-800 rounded-lg p-3 text-left hover:bg-gray-700"
+              >
+                <Image
+                  src={track.artwork_url}
+                  alt={track.track_name}
+                  width={200}
+                  height={200}
+                  className="w-full aspect-square object-cover rounded mb-2"
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)